components/AskAboutGroupModal.js

```diff
@@ -32,80 +31,40 @@ function reducer(state, action) {
         ...state,
         success: action.value,
       };
-    case 'setFirstName':
-      return {
-        ...state,
-        firstName: action.value,
-      };
-    case 'setLastName':
-      return {
-        ...state,
-        lastName: action.value,
-      };
-    case 'setEmail':
-      return {
-        ...state,
-        email: action.value,
-      };
-    case 'setQuestion':
-      return {
-        ...state,
-        question: action.value,
-      };
     default:
       throw new Error();
   }
 }
 
-export default (props) => {
-  const [
-    { loading, success, firstName, lastName, email, question },
-    dispatch,
-  ] = useReducer(reducer, initialState);
-  const dropdownAlertRef = useRef(null);
+export default function AskAboutGroupModal({ groupID, title, showSuccess }) {
+  const [{ loading, success }, dispatch] = useReducer(reducer, initialState);
 
   const handleOpenModal = () => {
     Amplitude.logEventWithProperties('OPEN Group Ask Question', {
-      group: props.title,
+      group: title,
     });
   };
 
-  const handleAsk = () => {
-    if (!firstName || !lastName || !email || !question) {
-      return dropdownAlertRef.current.alertWithType(
```

Comment:
Formik includes validation, so the validation error messages are baked into the new `<TextInput>` component